tests/Api/ProductTest.php

```diff
@@ -30,4 +30,15 @@ public function test_can_call_magento_api_products_show()
 
         $this->assertTrue($api->ok());
     }
+
+    public function test_can_edit_product()
+    {
+        Http::fake([
+            '*rest/default/V1/products/foo' => Http::response([], 200),
```

Comment:
```suggestion
            '*rest/default/V1/products/foo*' => Http::response([], 200),
```
For tests to pass--
I think GH Actions has some trailing slash or something going on. :)